src/overlay.js

```diff
@@ -303,7 +311,7 @@
                         style[transformProp] = "";
                     }
                 }
-                style.display = 'block';
+                style.display = 'flex';
```

Comment:
Nice, this should allow an overlay to have [Flex Grow](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow) equal to 1 and fill the outerWrapper size.